src/main/java/dk/kb/present/DSCollection.java

```diff
@@ -97,6 +104,7 @@ public DSCollection(YAML conf, StorageHandler storageHandler) {
             prefix = conf.getString(PREFIX_KEY);
             description = conf.getString(DESCRIPTION_KEY, null);
             storage = storageHandler.getStorage(conf.getString(STORAGE_KEY, null)); // null means default storage
+            getRecordEndpoint = conf.getString(GET_RECORD_ENDPOINT_KEY);
```

Comment:
This makes the get-record-endpoint mandatory for all collections, but we're not using it for anything for most of them (all the picture stuff). How about making it optional by stating `null` as a default and check for it where used?